Documentation/boards/raspberrypi.md

```diff
@@ -21,14 +21,27 @@ For access to terminal over serial console, add `console=ttyAMA0,115200` to `cmd
 
 ## I2C
 
-Add `dtparam=i2c1=on` and `dtparam=i2c_arm=on` to `config.txt`. After that we create a module file on host with [config usb stick][config] or direct into `/etc/modules-load.d`. 
+Add `dtparam=i2c1=on` and `dtparam=i2c_arm=on` to `config.txt`. After that we create a module file on host with [config usb stick][config] or direct into `/etc/modules-load.d`.
 
 rpi-i2c.conf:
 ```
 i2c-dev
 i2c-bcm2708
 ```
 
+## USB Boot
+
+USB mass storage boot is available on Raspberry Pi 3B, 3B+, 3A+, and 2B v1.2.
```

Comment:
also pi 4?

Reply:
Not at the moment it seems: 
https://www.raspberrypi.org/forums/viewtopic.php?t=243995
https://www.raspberrypi.org/documentation/hardware/raspberrypi/booteeprom.md